### PR TITLE
Update reinforced token generation with operation description

### PR DESCRIPTION
**Pull request by elon0823**
#### Operation Description Update
- Support reinforced token generation with `operation description` which was the `operation_name`.
- Update `auth` API body data schema to match operation description structure.
- Add test case for authorization with changed `operation description`.
- Update READE for `operation description` field details.

#### Token Removal
- Add token remove function to delete token from redis after the operation has been performed.

#### Extra
- Add missing `request` module on requirements.py.
- Fix misspelled url on Config.

### CODE REVIEW

File: auth/tests.py [hunk 1]
```diff
@@ -7,17 +7,37 @@
...
+        'X-Signature': signature,
+    }
 
 
 class AuthTest(unittest.TestCase):
 
     def create_app(self):
         return create_app_base(
-            SQLALCHEMY_DATABASE_URI='sqlite:////Users/jayden/Works/amo-storage/auth_test.db',
+            SQLALCHEMY_DATABASE_URI='sqlite://///Users/elon/Documents/develop/amo-storage/auth_test.db',
```
Comment (Lbird) [inline auth/tests.py]: Make it configurable

File: config.py [hunk 2]
```diff
@@ -6,7 +6,7 @@ class AmoStorageConfig(object):
 
     # SQLite Configurations
     SQLALCHEMY_TRACK_MODIFICATIONS = False
-    SQLALCHEMY_DATABASE_URI = 'sqlite:///tmp/amo_storage.db'
+    SQLALCHEMY_DATABASE_URI = 'sqlite:////tmp/amo_storage.db'
```
Comment (Lbird) [inline config.py]: Make it configurable

File: parcels/tests.py [hunk 3]
```diff
@@ -56,7 +56,7 @@ class ParcelsTest(unittest.TestCase):
 
     def create_app(self):
         return create_app_base(
-            SQLALCHEMY_DATABASE_URI='sqlite:////Users/jayden/Works/amo-storage/parcels_test.db',
+            SQLALCHEMY_DATABASE_URI='sqlite:////Users/elon/Documents/develop/amo-storage/parcels_test.db',
```
Comment (Lbird) [inline parcels/tests.py]: Make it configurable

File: config.py (hunk 2 above)
Comment (code-to-gold) [reply config.py]: It's already configurable. User can change DATABASE URI on config.py as above. I have no idea what you're talking about.

File: auth/tests.py (hunk 1 above)
Comment (code-to-gold) [reply auth/tests.py]: This line is for overriding default configurations to run unit test.

Comment (Lbird) [reply auth/tests.py]: ok, i see.

File: config.py (hunk 2 above)
Comment (Lbird) [reply config.py]: Hmmm... I didn't imagine this file is a user configuration file. I thought this is just for default config values. I expected more popular type for a user configuration file, e.g. ini, yaml, json, config, and so on. Are you planning to keep this format? In that case, a user surely needs a configuration guide to write a *class* to config something. In real env, an administrator may not know anything about python grammar.

Comment (code-to-gold) [reply config.py]: We gonna add configuration guide. This config.py is pythonic style to handle configurations(using built-in data structure). If you want external file like .ini or .json, we will reflect it.

Comment (Lbird) [reply config.py]: It's much simpler to use ini file for users. Please make a new PR for the job. I'll merge this in this state.

**Pull request merged**